src/legacy/state/orders/updaters/UnfillableOrdersUpdater.ts

```diff
@@ -212,7 +179,7 @@ export function UnfillableOrdersUpdater(): null {
       isUpdating.current = false
       console.debug(`[UnfillableOrdersUpdater] Checked pending orders in ${Date.now() - startTime}ms`)
     }
-  }, [account, chainId, strategy, updateIsUnfillableFlag, isWindowVisible, updatePendingOrderPrices])
+  }, [account, chainId, strategy, updateIsUnfillableFlag, isWindowVisible, balances, updatePendingOrderPrices])
```

Comment:
Doesn't it create excessive triggers? There was a problem with re-renders with `useBalancesAndAllowances`

Reply:
Just tested it, works good!